test/suites/expiration/suite_test.go

```diff
@@ -102,12 +98,6 @@ var _ = Describe("Expiration", func() {
 			g.Expect(ok).To(BeTrue())
 		}).Should(Succeed())
 
-		// Set the expireAfter to "Never" to make sure new node isn't deleted
-		// This is CRITICAL since it prevents nodes that are immediately spun up from immediately being expired and
-		// racing at the end of the E2E test, leaking node resources into subsequent tests
-		nodePool.Spec.Template.Spec.ExpireAfter.Duration = nil
```

Comment:
The more I think about it, we should set the limit on the nodepool to zero after one expiration happens. That way we won't delete our current nodeclaims, but we also won't scale up either. This is important to prevent the case of leaked resources